src/entities/notification.rs

```diff
@@ -27,6 +27,7 @@ pub enum NotificationType {
     PollExpired,
     Status,
     EmojiReaction,
+    Update,
```

Comment:
Why did you mix this into this PR? Could you separate it into a separate PR?

Reply:
Done